src/Schema/SchemaContainer.php

```diff
@@ -171,10 +169,33 @@ protected function hasPath(Connection $connection, $parent)
     }
 
     /**
+     * Add connection to collection.
+     *
+     * @param  string $name
+     * @param  string $namespace
+     * @return Field
+     */
+    public function connection($name, $namespace)
+    {
+        $edgeType = $this->createField($name.'Edge', $namespace);
+
+        $this->types->push($edgeType);
+
+        $connectionType = $this->createField($name.'Connection', $namespace);
```

Comment:
It would still generate both the ConnectionType and the EdgeType. I got a little tripped up because I registered ```Relay::connection('posts', 'ClassName')``` but when I went to use it (```GraphQL::type('posts')```) I got an error stating that the type wasn't registered. I didn't realize I needed to use ```GraphQL::type('postsConnection')``` instead.

Since we use the same name for Types, Queries and Mutations, I was thinking we would want to use the same name for connections. I understand what you're trying to do here though, so I could stick with what you originally had and just notate the Wiki.